deployments-unknown.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I'm confirming I diffed this file against the unknown section of the original deployments.json and found no changes.